### ISSUE TITLE
a problem about Mem2Seq.py

### ISSUE THREAD

**Issue by bing-95** [HLTCHKUST/Mem2Seq#8]
Hi, in the Mem2Seq.py, I have a question about the GRU's hidden layer. 
In the DecoderMemNN _init_ function, the code `self.gru = nn.GRU(embedding_dim, embedding_dim, dropout=dropout)` doesn't define the GRU's layer_num, so does it have a default one hidden layer?
And in the train_batch and evaluate_batch function, the code `decoder_hidden = self.encoder(input_batches).unsqueeze(0)` also implies the GRU has one hidden layer.
So, I think, during training and evaluating, the GRU always has only one hidden layer regardless of the args['layer']. Should the code be modified? 

Look forward to your reply, thanks.

**Comment by jasonwu0731** [comment]
Yes for all the experiments we did are using one layer for the GRU. The args["layer"] we set is for the hyperparameter in the memory network setting, that is, it is for the number of "hops". 

Please feel free to try a different number of RNN layers to see if the results can be improved. :)

**Comment by bing-95** [comment]
> Yes for all the experiments we did are using one layer for the GRU. The args["layer"] we set is for the hyperparameter in the memory network setting, that is, it is for the number of "hops".
> 
> Please feel free to try a different number of RNN layers to see if the results can be improved. :)

OK, I understantd. Thank you~ :)
